ee/codegen/src/project.ts

```diff
@@ -191,6 +191,26 @@ ${errors.slice(0, 3).map((err) => {
     }
   }
 
+  // preprocessWorkflowConfig is used to modify the workflow config before it is used to generate the project.
+  // - Changes the combinator of any AND conditions to OR conditions, this should happen rarely and only for
+  //   resiliency reasons.
```

Comment:
[nit] I'd give this a try from `ee/codegen/src/serializers/vellum.ts` on `NodeInputSerializer`:

```typescript
    combinator: undiscriminatedUnionSchema([
      stringLiteralSchema("OR"),
      stringLiteralSchema("AND"),
    ]).transform({
      transform: () => "OR" as const,
      untransform: () => "OR" as const,
    }),
```

It allows us to not need to do a preprocess step on unknown jsons, keeping everything strictly typed

Reply:
you'll also need to update the `combinator` annotation for `NodeInputSerializer.Raw`